frontend/imports/utils/conversion.js

```diff
@@ -1,15 +1,36 @@
-import { Dapple, web3 } from 'meteor/makerotc:dapple';
+import { Dapple } from 'meteor/makerotc:dapple';
 import { BigNumber } from 'meteor/ethereum:web3';
 
-export default function convertToWei(amount, token) {
-  const tokenSpecs = Dapple.getTokenSpecs(token);
-  const precision = tokenSpecs !== undefined ? tokenSpecs.precision : 18;
-  if (precision === 18) {
-    return web3.toWei(amount);
+export function convertToAbsolute(amount, token) {
```

Comment:
You are right, "convertToAbsolute" was not the correct name. I renamed to "convertToTokenPrecision".

About wei, as far as I understand it just multiplies the value by 10^18, so felt no need to keep the special case of using `toWei` when  `=== 18`.
Then in other cases had to replace `web3.toWei` because was not correctly used.
For example in newallowance.js it was converting the value of the input using `web3.toWei` when it should depend on the precision of the token. In case of W-ETH or MKR it will still multiply by 10^18, but for DGD, it will need 10^9.